Documentation/tutorial/intro_toc.md

```diff
@@ -64,34 +64,60 @@ Property Name | Type              | Description
 ------------- | ----------------- | ---------------------------
 *name*        | string            | Specifies the title of the *TOC Item*.
 *href*        | string            | Specifies the hyperlink of the *TOC Item*.
-*homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
-*uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.
-*homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
-*items*       | *TOC Item Object* | Specifies the children *TOC Items*s of current *TOC Item*.
+*items*       | *TOC Item Object* | Specifies the children *TOC Item*s of current *TOC Item*.
+
+**Advanced**: These properties is useful when a TOC links to or reference to another TOC, or links to a uid.
+
+Property Name                     | Type              | Description
+--------------------------------- | ----------------- | ---------------------------
+*tocHref*                         | string            | Specifies another TOC file to be expanded in the currrent *TOC Item*.
+*topicHref*                       | string            | Specifies the homepage of the referenced *TOC Item*. It is useful when *tocHref* is used.
+*topicUid*                        | string            | Specifies the `uid` of the homepage or the referenced file. If the value is set, it overwrites the value of *topicHref*.
+*homepage*                        | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
+~~*uid*~~ **Deprecated**          | string            | ~~Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.~~ Use *topicUid* instead.
+~~*homepageUid*~~ **Deprecated**  | string            | ~~Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.~~ Use *topicUid* instead.
 
 Relative path in detail
 ---------------
 If a *TOC Item* is linking to some relative path, there are three cases:
 
-1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
+1. **Deprecated** ~~Linking to another *TOC File*, for example, `href: examples/toc.md`.~~
```

Comment:
>**Deprecated**  [](start = 3, length = 15)

a bit confusing since we changed the way to support this option, but this option is still available